tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -399,7 +399,7 @@ public void getCapacityBytes() {
   @Test
   public void lastModificationTimeCompleteFile() throws Exception {
     long fileId = mFsMaster.createFile(new AlluxioURI("/testFile"), CreateFileOptions.defaults());
-    long opTimeMs = TEST_CURRENT_TIME;
+    long opTimeMs = System.currentTimeMillis() + TEST_TIME_OFFSET_MS;
```

Comment:
I could use max long value here instead if you prefer. The change is because we need to have an operation time which is later than the creation time since we no longer update the last mod time if it is stale during rename (due to allowing for concurrent renames).